wp-content/themes/core/assets/js/src/theme/core/components.js

```diff
@@ -11,6 +11,7 @@ import share from 'components/share';
 import slider from 'components/slider';
 import tabs from 'components/tabs';
 import video from 'components/video';
+import gallery from 'components/blocks/gallery_grid';
```

Comment:
Let's keep the name name consistent here (`blockGalleryGrid` or similar), since there's a good chance we'll have other "gallery" modules to import.